cmd/nerdctl/builder_build_linux_test.go

```diff
@@ -0,0 +1,57 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/containerd/nerdctl/v2/pkg/testutil"
+	"gotest.tools/v3/assert"
+)
+
+func TestBuildContextWithOCILayout(t *testing.T) {
+	// Docker driver does not support OCI exporter.
```

Comment:
`docker buildx build` supports it though

Reply:
Thanks @AkihiroSuda, that helped me figure out the issue. I needed to setup a docker-container builder in order to be able to build the OCI tarball.